src/home/saga.ts

```diff
@@ -63,6 +64,7 @@ export function* watchRefreshBalances() {
       createFiatConnectTransferCompleted.type,
       depositTransactionSucceeded.type,
       swapSuccess.type,
+      Web3Actions.DEMO_MODE_TOGGLED,
```

Comment:
seems sensible to refresh the balances when the demo mode is toggled. the balances sagas are terminating early if there is no address anyway